demo/site/src/pages/[[...path]].tsx

```diff
@@ -53,67 +48,41 @@ const pageTypes = {
     },
 };
 
-export const getStaticProps: GetStaticProps<PageUniversalProps> = async (context) => {
-    const getUniversalProps = createGetUniversalProps();
-    return getUniversalProps(context);
-};
-
-interface CreateGetUniversalPropsOptions {
-    includeInvisiblePages?: boolean;
-    includeInvisibleBlocks?: boolean;
-    previewDamUrls?: boolean;
-}
-
-// a function to create a universal function which can be used as getStaticProps or getServerSideProps (preview)
-export function createGetUniversalProps({
-    includeInvisiblePages = false,
-    includeInvisibleBlocks = false,
-    previewDamUrls = false,
-}: CreateGetUniversalPropsOptions = {}) {
-    return async function getUniversalProps<Context extends GetStaticPropsContext | GetServerSidePropsContext>({
-        params,
-        locale = defaultLanguage,
-    }: Context): Promise<
-        Context extends GetStaticPropsContext ? GetStaticPropsResult<PageUniversalProps> : GetServerSidePropsResult<PageUniversalProps>
-    > {
-        const path = params?.path ?? "";
-        const contentScope = { domain, language: locale };
-
-        //fetch pageType
-        const data = await createGraphQLClient({ includeInvisiblePages, includeInvisibleBlocks, previewDamUrls }).request<
-            GQLPageTypeQuery,
-            GQLPageTypeQueryVariables
-        >(pageTypeQuery, {
-            path: `/${Array.isArray(path) ? path.join("/") : path}`,
-            contentScope,
-        });
-        if (!data.pageTreeNodeByPath?.documentType) {
-            // eslint-disable-next-line no-console
-            console.log("got no data from api", data, path);
-            return { notFound: true };
-        }
-        const pageId = data.pageTreeNodeByPath.id;
+export const getStaticProps: GetStaticProps<PageUniversalProps, ParsedUrlQuery, PreviewData> = async ({
```

Comment:
great that we don't need this "Universal" thing anymore!

Please also rename `PageUniversalProps` accordingly